snuba/datasets/events.py

```diff
@@ -228,8 +228,8 @@ def column_expr(self, column_name, body):
             return self._tag_expr(column_name)
         elif column_name in ['tags_key', 'tags_value']:
             return self._tags_expr(column_name, body)
-        elif column_name == 'issue':
-            return 'group_id'
+        elif column_name == 'issue' or column_name == 'group_id':
```

Comment:
By the way, why do we expose two different column names here for what appears to be the same thing?

Reply:
I suspect this is for historical reasons, but looking at queries coming from sentry on Snuba tests, they all seem to use 'issue' instead of group_id